.travis.yml

```diff
@@ -22,6 +25,13 @@ jobs:
       julia: 1.0
       os: linux
       script:
+        # html
         - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
         - julia --project=docs/ docs/make.jl
-      after_success: skip
+        # pdf
+        # - julia --project=docs/pdf/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
```

Comment:
Was running into https://github.com/JuliaLang/Pkg.jl/issues/913 here so yey for shooting myself in the foot.